litestar_saq/info.py

```diff
@@ -11,6 +11,6 @@
 async def job(queue: Queue, job_id: str) -> Job:
     job = await queue._get_job_by_id(job_id)  # noqa: SLF001
     if not job:
-        msg = "Could not find job ID %s"
-        raise LitestarSaqError(msg, job_id)
+        msg = f"Could not find job ID {job_id}"
+        raise NotFoundException(msg)
```

Comment:
I raised a Litestar exception here as only the route handlers depended on this (`info.job`) function. I am not sure if you would prefer to have it this way.